types/sinch-rtc/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
```

Comment:
This needs to be removed and error needs to be fixed

Reply:
Fixed